toolchain/parser/parser_context.cpp

```diff
@@ -40,23 +40,23 @@ static auto operator<<(llvm::raw_ostream& out, RelativeLocation loc)
   return out;
 }
 
-Context::Context(Tree& tree, TokenizedBuffer& tokens,
-                 TokenDiagnosticEmitter& emitter,
+Context::Context(Tree& tree, Lex::TokenizedBuffer& tokens,
+                 Lex::TokenDiagnosticEmitter& emitter,
                  llvm::raw_ostream* vlog_stream)
     : tree_(&tree),
       tokens_(&tokens),
       emitter_(&emitter),
       vlog_stream_(vlog_stream),
       position_(tokens_->tokens().begin()),
       end_(tokens_->tokens().end()) {
-  CARBON_CHECK(position_ != end_) << "Empty TokenizedBuffer";
+  CARBON_CHECK(position_ != end_) << "Empty Lex::TokenizedBuffer";
```

Comment:
`Lex::` seems unnecessary here (sorry, GH won't let me express that as a suggestion)